sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergWriteSchemaTransformProvider.java

```diff
@@ -17,13 +17,21 @@
  */
 package org.apache.beam.sdk.io.iceberg;
 
+import static org.apache.beam.sdk.io.iceberg.IcebergWriteSchemaTransformProvider.Configuration;
+
 import com.google.auto.service.AutoService;
+import com.google.auto.value.AutoValue;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
```

Comment:
Use checkerframework `Nullable` annotation (unless underlying library like Avro requires you to use theirs). I know it seems like an annotation wouldn't matter, but they have different capabilities in some cases so we standardize on the one that works everywhere it needs to.

Reply:
Done